modules/touch/touchmodule.cpp

```diff
@@ -35,26 +35,37 @@
 using namespace TUIO;
 
 namespace {
-    constexpr openspace::properties::Property::PropertyInfo TouchActiveInfo = {
-        "TouchActive",
-        "True if we want to use touch input as 3D navigation",
-        "Use this if we want to turn on or off Touch input navigation. "
-        "Disabling this will reset all current touch inputs to the navigation."
+    constexpr openspace::properties::Property::PropertyInfo EnableTouchInfo = {
+        "EnableTouchInteraction",
+        "Enable Touch Interaction",
+        "Use this property to turn on/off touch input navigation in the 3D scene. "
+        "Disabling will reset all current touch inputs to the navigation."
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo EventsInfo = {
+        "DetectedTouchEvent",
+        "Detected Touch Event",
+        "True when there is an active touch event",
+        openspace::properties::Property::Visibility::Hidden
     };
 }
 namespace openspace {
 
 TouchModule::TouchModule()
     : OpenSpaceModule("Touch")
-    , _touchActive(TouchActiveInfo, true)
+    , _touchIsEnabled(EnableTouchInfo, true)
+    , _hasActiveTouchEvent(EventsInfo, false)
 {
     addPropertySubOwner(_touch);
     addPropertySubOwner(_markers);
-    addProperty(_touchActive);
-    _touchActive.onChange([&] {
+    addProperty(_touchIsEnabled);
+    _touchIsEnabled.onChange([&] {
```

Comment:
-> `[&]() {`

() was missing in the old version already

Reply:
Good 👁️